### PR TITLE
Ex3

### PR DESCRIPTION
**Pull request by zyahav**

### CODE REVIEW

File: ex3B/ex3Spec.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var ex3 = require('./ex3');
+
+describe("containsValue", function() {
+
+	it("shoud not find the number 8 in array", function() {
```
Comment (myok12) [inline ex3B/ex3Spec.js]: :%s/shoud/should/g

Or: typo in shoud; supposed to be should.

Comment (myok12) [inline ex3B/ex3Spec.js]: Same below

File: ex3B/ex3.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/**
+ * Write a function that is given an array of numbers and a number and returns
+ * whether the number exists within the array.
+ *
+ * For example:
+ * containsValue([1,3,5,7,9], 8) === false;
+ * containsValue([1,3,5,7,9], 7) === true;
+ * containsValue([9,7,5,3,1], 7) === true;
+ */
+function containsValue(arrayList, searchValue) {
```
Comment (myok12) [inline ex3B/ex3.js]: In computer science: An array is a continuos piece of memory that contains some values. A list is some values in some order. As you can see "Array""List" doesn't necessarily reflect what you had in mind. How about array, arr, list, valueArr, valueList, or maybe just values.

File: ex3B/ex3Spec.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+		expect(ex3.containsValue([1,3,5,7,9], 8)).toBeFalsy();
+	});
+
+	it("shoud find the number 7 in array", function() {
+		expect(ex3.containsValue([1,3,5,7,9], 7)).toBeTruthy();
+	});
+
+	it("shoud find the number 7 in array", function() {
```
Comment (myok12) [inline ex3B/ex3Spec.js]: should find a number when the input is not in order.

File: ex3B/ex3Spec.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+var ex3 = require('./ex3');
+
+describe("containsValue", function() {
+
+	it("shoud not find the number 8 in array", function() {
+		expect(ex3.containsValue([1,3,5,7,9], 8)).toBeFalsy();
+	});
+
+	it("shoud find the number 7 in array", function() {
```
Comment (myok12) [inline ex3B/ex3Spec.js]: should find a number when the input is in order.

File: ex3B/ex3Spec.js (hunk 1 above)
Comment (myok12) [inline ex3B/ex3Spec.js]: Should correctly answer when the value isn't found.

Comment (zyahav) [inline ex3B/ex3Spec.js]: done

File: ex3B/ex3Spec.js (hunk 4 above)
Comment (zyahav) [inline ex3B/ex3Spec.js]: done

File: ex3B/ex3Spec.js (hunk 3 above)
Comment (zyahav) [inline ex3B/ex3Spec.js]: done

File: ex3B/ex3.js (hunk 2 above)
Comment (zyahav) [inline ex3B/ex3.js]: done

**Pull request merged**